htmd/queues/lsfqueue.py

```diff
@@ -68,14 +68,19 @@ def __init__(self, _configapp=None):
         self._arg('ncpu', 'int', 'Number of CPUs to use for a single job', self._defaults['ncpu'],
                   val.Number(int, '0POS'))
         self._arg('memory', 'int', 'Amount of memory per job (MB)', self._defaults['memory'], val.Number(int, '0POS'))
-        self._arg('walltime', 'int', 'Job timeout (hour:min or min)', self._defaults['walltime'], val.Number(int, '0POS'))
+        self._arg('walltime', 'int', 'Job timeout (hour:min or min)', self._defaults['walltime'],
+                  val.Number(int, '0POS'))
         self._arg('resources', 'list', 'Resources of the queue', self._defaults['resources'], val.String(), nargs='*')
-        self._arg('environment', 'list', 'Things to run before the job (sourcing envs).', self._defaults['environment'],
-                  val.String(), nargs='*')
+        self._cmdDeprecated('environment', 'prerun')
```

Comment:
Shouldn't it be `'environment', 'envvars'`?

Reply:
In LsfQueue, the `environment` meant commands to run before running the application (while in Slurm meant passing environment variables). That was the inconsistency and why a deprecation of the name was needed, and two new names were needed.

Reply:
Ok great!